### DISCUSSION TITLE
How can I hide the console  window in windows?

### DISCUSSION THREAD

**Discussion by Taiyangqihuo** [slint-ui/slint#4307] [Q&A] [answered]
I write a simply native app demo in windows and use winit as the backend. After I compiled this program , when I execute it , there will be a console  window behind the main window. How can I hide this console window? Should I use a winapi crate or other crates to do this?

**Comment by ogoffart** [comment, marked as answer]
You can.use  `#![windows_subsystem = "windows"]` in the main.rs

See https://doc.rust-lang.org/reference/runtime.html#the-windows_subsystem-attribute
And https://github.com/slint-ui/slint/issues/3235

**Reply by Taiyangqihuo** [reply]
Thanks a lot!
